middleware/webhooks.js

```diff
@@ -104,6 +103,13 @@ async function parseUserAction(entry) {
   } else if (event.referral) {
     // If the user has interacted with the bot before and they're following a
     // referral link, this statement will fire
+    if (isValidCommand(referralCommand) === false) {
+      return (parsed_data = {
+        ...parsed_data,
+        ...queryStringToObject('command=start_book,book_id=1'),
+        type: 'referral'
+      });
+    }
```

Comment:
I was typing a note here saying we should do the check for a valid command inside of CommandList instead of here but that made me realize to keep our current separation of concerns, it makes sense to do the default command in CL instead of here anyway, so you were given bad instruction from the get-go (insert facepalm image).

So let's roll with this for now, because this looks like it'll work fine, and we can update it later.

Reply:
Yea, good idea. 
I'll create a card for that change and get it tomorrow.